paper/paper.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Osprey: Hyperparameter Optimization for Machine Learning'
+tags:
+  - Python
+  - optimization
+  - cross-validation
+  - machine learning
```

Comment:
Can you change my affiliation to Stanford?

Sent from my iPhone

> On Jul 1, 2016, at 3:31 PM, msultan <notifications@github.com> wrote:
> 
> In paper/paper.md:
> 
> > @@ -0,0 +1,57 @@
> > +---
> > +title: 'Osprey: Hyperparameter Optimization for Machine Learning'
> > +tags:
> > +  - Python
> > +  - optimization
> > +  - cross-validation
> > +  - machine learning
> should we tag scikit-learn here too?
> 
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>